docs/contributing/coding-standards/components.md

```diff
@@ -10,19 +10,19 @@ Generally, folder and file names should be singular, for example ‘accordion’
 
 When creating your component, you should create the following files in the component’s folder:
 
-      - `README.md`
-      - `_[component-name].scss`
-      - `_index.scss`
-      - `[component-name].yaml`
-      - `macro.njk`
-      - `template.njk`
-      - `template.test.js`
+- `README.md` - Light documentation pointing to installation instructions and component documentation on <https://design-system.service.gov.uk/>
+- `_[component-name].scss` - An SCSS file to generate the styles just for the component, importing the variables, functions and mixins from `../../base` and the styles of the component (`./index.scss`).
+- `_index.scss` - The actual styles for the component, that can be imported either on their own by `[component-name].scss` or alongside other components in `components/_all.scss` (or by the consuming application)
+- `[component-name].yaml` - Lists the options expected by the component, used for display in the documentation. It also contains the option samples used to generate the examples in the documentation, as well as the [fixtures for testing alternative implementations of the design system](https://frontend.design-system.service.gov.uk/testing-your-html/).
+- `macro.njk` - The main entry point for rendering the component, provides a `govuk[ComponentName](params)` macro, delegating render to the `template.njk` file
+- `template.njk` - The template used for rendering the component using any `params` provided to the macro
+- `template.test.js` - Tests to ensure the component renders as intended with its various options
 
 If your component uses JavaScript, you must also create the following files in the component’s folder:
 
-      - `[component-name].mjs`
-      - `[component-name].unit.test.mjs`
-      - `[component-name].test.js`
+- `[component-name].mjs` - A JavaScript module with the implementation of any behaviour needed by the component. See the [JavaScript documentation]('./js.md#skeleton) for a skeleton and more details on that file's structure
+- `[component-name].unit.test.mjs` - Unit tests to verify any lower-level logic used by the component individually
```

Comment:
'individually' is an orphan in this sentence. Is it still accurate to say 'Does individual unit tests to verify...' ?

Reply:
Could maybe reword as "Unit tests to verify any component-specific lower-level logic"?

Basically, these are testing behaviour used internally by this component only.

Reply:
@domoscargin That sounds much better than the way I phrased it initially 😄

Reply:
Yup, that's fab.